soc/silabs/silabs_siwx91x/siwg917/nwp.c

```diff
@@ -86,11 +91,93 @@ static int siwg917_nwp_init(void)
 					SL_SI91X_BLE_ENABLE_ADV_EXTN |
 					SL_SI91X_BLE_AE_MAX_ADV_SETS(RSI_BLE_AE_MAX_ADV_SETS);
 #endif
+#endif /* WIFI_SILABS_SIWX91X_AP_MODE */
+
+	if (wifi_oper_mode == SL_SI91X_CLIENT_MODE) {
+		cfg->oper_mode = SL_SI91X_CLIENT_MODE;
+		cfg->ext_tcp_ip_feature_bit_map = SL_SI91X_CONFIG_FEAT_EXTENSION_VALID;
```

Comment:
This override previous assignment of `cfg->ext_tcp_ip_feature_bit_map`.

BTW, I thought `SL_SI91X_CONFIG_FEAT_EXTENSION_VALID` was a no-op while `config_feature_bit_map` was empty. If I am right, it would be easier to always set it, no?